Makefile

```diff
@@ -89,3 +89,10 @@ $(BIN)/golangci-lint: Makefile
 $(BIN)/protoc-gen-go: Makefile
 	@mkdir -p $(@D)
 	GOBIN=$(abspath $(@D)) $(GO) install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.1
+
+services.bin: $(BIN)/buf
+	$(BIN)/buf build --as-file-descriptor-set --output $(@F) \
+		buf.build/grpc/grpc:26635376b3f47a11126a0f4b4b5b6de7fe5a074a \
+		--type grpc.health.v1.Health \
+		--type grpc.reflection.v1.ServerReflection \
+		--type grpc.reflection.v1alpha.ServerReflection
```

Comment:
nit: can we keep a trailing newline here, so `cat Makefile` displays properly?

Reply:
Done. 0df528356d1aca3c1b57b7cb9da1b4d2f907016e